### ISSUE TITLE
Accessibility Audit

### ISSUE THREAD

**Issue by tonyedwardspz** [south-west-communities/website#6]
Audit the website for accessibility best practices.

**Comment by SteveALee** [comment]
I'll do a mini audit ASAP

**Comment by SteveALee** [comment]
Can you assign it to me?

**Comment by tonyedwardspz** [comment]
Looks like you need to be a part of the org @SteveALee ???

Just sent an invite. Will assign once you are. Appreciate the help.

**Comment by SteveALee** [comment]
Hm, that makes sense I guess. Though sometimes you might want to assign to outside person

**Comment by SteveALee** [comment]
OK, using Deque's aXe tool for the automatic testing aspect it's not doing too bad.

For these pages:

* https://southwestcommunities.co.uk
* https://southwestcommunities.co.uk/communities/
* https://southwestcommunities.co.uk/calendar/
* https://southwestcommunities.co.uk/proposals/

These are the errors

* A bunch of colour contrast errors at AA - WCAG failure: SC 1.4.3
* Multiple links with same name -  WCAG failure: 2.4.4, 4.1.2 
* No use of  landmarks - WCAG Best practice

Total being 200

I can provide more info and/or a PR if you like?

**Comment by tonyedwardspz** [comment]
Happy to take a PR, but I know/can figure out how to fix the first two. Your shout.

No idea what a landmark is though (in the context of a website). Any pointers?

**Comment by SteveALee** [comment]
> No idea what a landmark is though (in the context of a website). Any pointers?

Sure - but just advisory. Landmarks let you structure your page - \<main> \<nav> \<section> \<aside> and help AT users move around and understand page (eg screen readers). This post looks reasonable - https://keithjgrant.com/posts/2018/03/html5-sectioning-and-landmark-elements/

I'm busy this week so may best leave PR if you are happy. You have better idea of your design concept.

**Comment by SteveALee** [comment]
Here's the full audit 
[out-notty.txt](https://github.com/our-communities/our-communities-website/files/2888624/out-notty.txt)

**Comment by SteveALee** [comment]
ps the node selectors in the output should work if pasted into the browser dev tools

**Comment by SteveALee** [comment]
And I'll try to do some user testing with a screen reader too.

**Comment by tonyedwardspz** [comment]
That's brilliant @SteveALee. Thanks for your time on this.

Landmarks make sense, just hadn't heard those elements referred to as that before :)

**Comment by SteveALee** [comment]
Great! They are pretty new really.

This might help with general a11y

https://www.w3.org/WAI/tips/designing/

**Comment by tonyedwardspz** [comment]
I've started using the chrome plugin `Accessibility Insights for Web`, which is built on top of axe-core.

It originally corresponding with your suggestions @SteveALee, but is now coming back as error free. Closing until other issues are highlighted.

**Comment by SteveALee** [comment]
Microsoft did a great job of collaboration there!
